HedgeModManager/Languages/hu-HU.xaml

```diff
@@ -0,0 +1,263 @@
+﻿<ResourceDictionary 
+  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+  xmlns:system="clr-namespace:System;assembly=mscorlib">
+
+    <!-- Common UI Strings -->
+    <system:String x:Key="CommonUIClose"   >Bezár</system:String>
+    <system:String x:Key="CommonUIOK"      >OK</system:String>
+    <system:String x:Key="CommonUICancel"  >Mégse</system:String>
+    <system:String x:Key="CommonUIYes"     >Igen</system:String>
+    <system:String x:Key="CommonUIBlocked" >Blokkolva</system:String>
+    <system:String x:Key="CommonUINo"      >Nem</system:String>
+    <system:String x:Key="CommonUIRetry"   >Újra</system:String>
+    <system:String x:Key="CommonUIUpdate"  >Frissítés</system:String>
+    <system:String x:Key="CommonUIDownload">Letöltés</system:String>
+    <system:String x:Key="CommonUIDelete"  >Törlés</system:String>
+    <system:String x:Key="CommonUINone"    >Nincs</system:String>
+    <system:String x:Key="CommonUIIgnore"  >Kihagyás</system:String>
+    <system:String x:Key="CommonUIWarning" >FIGYELMEZTETÉS</system:String>
+    <system:String x:Key="CommonUIError"   >HIBA</system:String>
+
+    <!-- Main UI Strings -->
+    <system:String x:Key="MainUIMods"          >Modok</system:String>
+    <system:String x:Key="MainUICodes"         >Kódok</system:String>
+    <system:String x:Key="MainUISettings"      >Beállítások</system:String>
+    <system:String x:Key="MainUIAddMod"        >Mod hozzáadása</system:String>
+    <system:String x:Key="MainUIRefreshModList">Lista frissítése</system:String>
+    <system:String x:Key="MainUISave"          >Mentés</system:String>
+    <system:String x:Key="MainUISaveAndPlay"   >Mentés és játék indítása</system:String>
```

Comment:
I changed to Mentés és indítás. Yes, you're right, even though "indítás" does not mean play directly, users will probably figure out that it will start the game.